src/dialect/snowflake.rs

```diff
@@ -76,12 +79,25 @@ impl Dialect for SnowflakeDialect {
             // possibly CREATE STAGE
             //[ OR  REPLACE ]
             let or_replace = parser.parse_keywords(&[Keyword::OR, Keyword::REPLACE]);
-            //[ TEMPORARY ]
-            let temporary = parser.parse_keyword(Keyword::TEMPORARY);
+            let mut global = None;
+            if parser.parse_keyword(Keyword::LOCAL) {
```

Comment:
wouldn't this permit something like `LOCAL GLOBAL` to parse? It seems like we should permit only one or the other